app/Console/Commands/PhotosAddedNotification.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Mail\PhotosAdded;
+use App\Models\Configs;
+use App\Models\Photo;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Storage;
+
+class PhotosAddedNotification extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'lychee:photos_added_notification';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Send email notifications for newly added photos';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return int
+	 */
+	public function handle()
+	{
+		$settings = Configs::get();
+
+		if ($settings['new_photos_notification']) {
+			$users = User::whereNotNull('email')->get();
+
+			foreach ($users as $user) {
+				$photos = [];
+
+				foreach ($user->unreadNotifications as $notification) {
+					$photo = Photo::find($notification->data['id']);
+
+					if ($photo && $photo->thumbUrl) {
+						if (!isset($photos[$photo->album_id])) {
+							$photos[$photo->album_id] = [
+								'name' => $photo->album->title,
+								'photos' => [],
+							];
+						}
+
+						logger(Storage::url(Photo::VARIANT_2_PATH_PREFIX[Photo::VARIANT_THUMB] . '/' . $photo->thumbUrl));
+
+						$photos[$photo->album_id]['photos'][$photo->id] = [
+							'thumb' => Storage::url(Photo::VARIANT_2_PATH_PREFIX[Photo::VARIANT_THUMB] . '/' . $photo->thumbUrl),
+							'link' => config('app.url') . '/r/' . $photo->album_id . '/' . $photo->id,
```

Comment:
@kamil4 it is redirect. :)
https://github.com/LycheeOrg/Lychee/blob/master/routes/web.php#L41

It is something I added a while back because of Instagram.
If you share something like https://photography.example.com/gallery#1234567 Instagram will break the URL into 2, one will be the link to the site, the other one will be seen as an instagram Hashtag. Stupid but that is the solution I came up with.

Reply:
Yeah, in retrospect the use of `#` wasn't a very good idea I guess... I realize we can't just unilaterally change it as it would break existing URLs that existing users may rely on, but what about making it an option, one that is on by default for new installations?

Reply:
The use of `#` was necessary because Lychee aimed to be a one page website. And yes I did not chose it, this is very **old** legacy. :|

(BTW it is dropped with livewire at the moment).